.github/workflows/ci.yml

```diff
@@ -441,6 +460,14 @@ jobs:
           - target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
     steps:
+      - name: Checkout sources
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - name: Set up gdal-bin and sqlite3-tools
+        if: runner.os == 'Linux'
+        run: sudo apt-get update && sudo apt-get install -y gdal-bin sqlite3-tools
```

Comment:
should gdal be also part of install dependencies recipe? just curious, ok if not

Reply:
Since our long term plan for this is to add a new crate to do the e2e tests in, lets keep it as is.